monitoring/scripts/track-cid-backfill/src/queries.ts

```diff
@@ -0,0 +1,97 @@
+import { QueryTypes } from "sequelize";
+import { sequelizeConn } from "./db";
+
+import fs from 'fs-extra'
+
+// Fetch all content nodes
+export async function getAllContentNodes(): Promise<
+  { spid: number; endpoint: string }[]
+> {
+  const endpointsResp: unknown[] = await sequelizeConn.query(
+    `
+    SELECT cnode_sp_id as spid, endpoint
+    FROM ursm_content_nodes
```

Comment:
i tagged you in slack discussion but this table is deprecated so will need to change
this query will be incorrect for prod

Reply:
it might honestly be easier to just hardcode this array into the code for prod runs rather than trying to make the code change

Reply:
Makes sense. Hardcoding the array is definitely the move cc @sddioulde

Reply:
sounds good to me